### ISSUE TITLE
[Discussion] 3D Unit healthbars

### ISSUE THREAD

**Issue by cbscribe** [kidscancode/godot_recipes#35]
Discussion for http://godotrecipes.com/3d/healthbars/

**Comment by dirkk0** [comment]
The ZIP is not working as described.
To make it work, change HealthBar3D as follows:
- set Texture to 'New ViewportTexture' and select the existing Viewport
- delete the Material in Geometry
- enable the Billboard flag

**Comment by cbscribe** [comment]
Yes, this is due to a change in the recent version of Godot (addition of the Billboard flag). 

The text has been changed, but the zip needs to be updated.

**Comment by dirkk0** [comment]
I can send you my version, if that helps.

**Comment by Raywilleatchu** [comment]
I like the video and demonstration however after following along, I found that the viewport does not update the sprite3D very well. Its leaving me with this gray box over my head. The viewport seems to not be setting the texture correctly? idk if thats a godot thing or a me thing
